port/jassimp/jassimp-native/src/jassimp.cpp

```diff
@@ -5,12 +5,30 @@
 
 
 #ifdef JNI_LOG
+#ifdef ANDROID
```

Comment:
looks like i'm not the only one...
http://stackoverflow.com/questions/15328751/android-macro-suddenly-not-defined